xbmc/addons/kodi-dev-kit/include/kodi/c-api/filesystem.h

```diff
@@ -221,7 +221,7 @@ extern "C"
     uint64_t forward;
     unsigned int maxrate;
     unsigned int currate;
-    bool lowspeed;
+    unsigned int lowrate;
```

Comment:
By change here need `ADDON_INSTANCE_VERSION_VFS_MIN` and `ADDON_INSTANCE_VERSION_VFS` a version increase and rebuild of addons.
On this place https://github.com/xbmc/xbmc/blob/master/xbmc/addons/kodi-dev-kit/include/kodi/versions.h#L164-L165 and as 3.0.1 would be enough.

Reply:
Overseen, need then also here a increase https://github.com/xbmc/xbmc/blob/master/xbmc/addons/kodi-dev-kit/include/kodi/versions.h#L65-L66 of `ADDON_GLOBAL_VERSION_FILESYSTEM_MIN` and `ADDON_GLOBAL_VERSION_FILESYSTEM`, then much addons :sweat_smile:.